cmd/tke-installer/app/installer/installer.go

```diff
@@ -170,6 +177,27 @@ func (t *TKE) loadTKEData() error {
 }
 
 func (t *TKE) initSteps() {
+
+	// Expansion
+	t.steps = append(t.steps, []types.Handler{
+		{
+			Name: "Load Expansion Operator Image",
+			Func: t.loadOperatorImage,
+		},
+	}...)
+	t.steps = append(t.steps, []types.Handler{
+		{
+			Name: "Patch Hook Files",
+			Func: t.patchHookFiles,
+		},
+	}...)
+	t.steps = append(t.steps, []types.Handler{
+		{
+			Name: "Start Expansion Operator",
+			Func: t.startOperator,
+		},
+	}...)
+
```

Comment:
These are not implemented now, I think we should remove dead code, focus on work which have been implemented.